ramls/url.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "URL definition",
+  "type": "string",
+  "pattern": "^(?:http(s)?:\\/\\/)?[\\w.-]+(?:\\.[\\w\\.-]+)+[\\w\\-\\._~:/?#[\\]@!\\$&'\\(\\)\\*\\+,;=.]+$",
```

Comment:
I suggest to completely remove url.json. If there is a need for a validation pattern please create a separate Jira and a separate pull request because it needs further discussion of the requirements.

The regular expression pattern doesn't compile. `#[\\]` needs to be replaced by `#\\[\\]`.

After fixing the pattern there are several issues:

It takes 10 seconds to validate strings like `http://....................................Ø` because of [regular expression denial of service (ReDoS)](https://en.wikipedia.org/wiki/ReDoS); this is a security issue.

The pattern rejects these perfectly valid URLs:
* https://münster.de
* https://en.wikipedia.org/wiki/Ø
* https://en.wikipedia.org/wiki/%C3%98

Reply:
> I suggest to completely remove url.json
Good point julian, it looks like validation of URLs for all the cases is not strighforward task, so it will be better to avoid such vaidation and remains with just string parameter, I agree. Especially since validation isn't a first proirity behavior here.